modules/publication/help/publication.md

```diff
@@ -0,0 +1,16 @@
+# Publication
+
+Publication is intended for users to propose specific research projects
+and for PIs to review them. When proposing a project, PIs may also specify 
+other users to help with editing and maintaining their project proposals.
+
+
+Navigate to the "__Propose a Project__" tab for proposing research projects based on the data required. You may upload publication media, and allows study PIs to review the project 
+proposals.
+
+Specify which specific variables of 
```

Comment:
```suggestion
You should specify which specific variables of interest are relevant to your research so
that the reviewers can determine if they should grant access to the data.
```

The suggested change is to replace these 3 lines, but I can't do multiline suggestions. (I don't think the line about module scope is relevant to the users. They might want to know how they'll get the data, but we don't have any control over that so I can't think of any way to write general purpose documentation..)

Reply:
@driusan how about now?